house/views.py

```diff
@@ -1,11 +1,13 @@
-from house.constants import MINE_MINE_PAGE_ROUTE
-from django.shortcuts import render, get_object_or_404
-from house.constants import HOME_PAGE_ROUTE, GLOBAL_PAGE_ROUTE, GLOBAL_PAGE_CITY_DROPDOWN_ROUTE, LOGIN_PAGE_ROUTE
-from .forms import HouseForm, HouseIDForm
-from .helpers import _filter_houses_by_form
-from expenses.models import Expenses
-from django.http import HttpResponseRedirect
+from django.contrib.auth.decorators import login_required
 from django.core.exceptions import ValidationError
+from django.http import HttpResponseRedirect
+from django.shortcuts import render
+
```

Comment:
empty line to delete

Reply:
Its ok, distinct between project import and third-party importa